stack-packs/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/** @type {import('./index').Pack[]} */
+const stackPacks = [
+  require('./packs/wordpress'),
```

Comment:
```suggestion
  require('./packs/wordpress.js'),
```

Reply:
why .js?

Reply:
we've been slowly adding these to the source base. it more clearly calls out files vs npm modules. also that its js and not json.